### PR TITLE
Add using different installation profile and remove gif from windows.

### PR DESCRIPTION
**Pull request by kporras07**

### CODE REVIEW

**Comment by rigoucr** [comment]
@kporras07 you forgot about add the optional step into the installation guide, pls add it

File: docs/working-with-humpback/using-different-installation-profile.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Using different installation profile
+
+Humpback by default create it's own installation profile, that profile is stored in profiles folder and its content is symlinked to the folder `web/profiles/custom`. This profile is used by default to install the site.
+If you want to override this behavior and use a different install profile (custom or contrib), you need to place that profile in the right folder (profiles/custom) or by adding it to the project using `composer require` and ensuring it gets placed under web/profiles/contrib and then make some small changes to ensure this profile gets used:
+
+- Remove all config files under config/sync to ensure profile config is used
+- Change install profile on .ahoy/site.ahoy around line 57 `ahoy drush site-install <profile_name> --account-pass=admin -y`
+- Change install profile on settings/settings.php around line 81 `$settings['install_profile'] = <profile_name>;`
+- Change install profile in circleci deploy job install command around line 108 `terminus drush dccr2019.dev -- si <profile_name> --account-pass=admin -y`
```
Comment (rigoucr) [inline docs/working-with-humpback/using-different-installation-profile.md]: dccr2019

**Comment by kporras07** [comment]
@rigoucr what do you mean by "installation guide"? Isn't it https://github.com/humpbackdev/humpback-docs/pull/1/files#diff-e25c06996bc095597ef800c4b3a75c1aR61 ?

**Comment by rigoucr** [comment]
> @rigoucr what do you mean by "installation guide"? Isn't it https://github.com/humpbackdev/humpback-docs/pull/1/files#diff-e25c06996bc095597ef800c4b3a75c1aR61 ?

I was think In something more specific, but it's ok, you could leave it as is now

**Review by rigoucr — APPROVED** [review]

**Comment by rigoucr** [comment]
@kporras07 :shipit:

**Pull request merged**